blocks/columns/columns.js

```diff
@@ -1,3 +1,36 @@
+export function applySplitPercentages(block) {
+  const ratios = [];
+  for (let i = 0; i < block.classList.length; i += 1) {
+    const cls = block.classList[i];
+    if (cls.startsWith('split-')) {
+      const numbers = cls.substring(6);
+      numbers.split('-').forEach((n) => ratios.push(`${n}%`));
+      break;
+    }
+  }
+
+  if (ratios.length === 0) {
+    return;
+  }
+
+  let pctIdx = 0;
+
+  for (let i = 0; i < block.children.length; i += 1) {
+    if (block.children[i].localName === 'div') {
+      for (let j = 0; j < block.children[i].children.length; j += 1) {
+        if (block.children[i].children[j].localName === 'div') {
+          block.children[i].children[j].style.flexBasis = ratios[pctIdx];
+          pctIdx += 1;
+
+          if (pctIdx >= ratios.length) {
+            pctIdx = 0;
```

Comment:
What does this do? Do the percentage cyclically repeat?

Reply:
The flex-basis needs to be applied to every cell in the columns 'table'. If there are multiple rows it starts the next row at 0 again.